docs/methods/calibration.md

```diff
@@ -0,0 +1,102 @@
+# Calibration
+
+## Calibration and optimization
+The first step of every alphaDIA entails the calibration of the empirical or fully predicted spectral library. This step has two main purposes: 1. Removing the systematic deviation of observed and library values and 2. Optimizing the size of the search space to reflect the expected deviation from the library values. For DIA search this means calibration and optimization of retention time, ion mobility, precursor m/z and fragment m/z.
+
+AlphaDIA uses a iterative calibration schedule consisting of epochs. Every epoch, a defined number of precursors need to be identified at 1% FDR before calibration and optimization is performed and the next epoch is started (dashed green line). For the first epoch this will be `recalibration_target: 200` precursors, for the second `400` for the third `600` etc. Within each epoch, increasing number of batches of size `batch_size: 8000` are searched and scored and aggregated, following an exponential batch plan (blue).
```

Comment:
I think mixing the config parameter name with the default value makes things more implicit than necessary..

Suggestion:
The number of precursors for the `n`'th epoch is derived from the parameter `recalibration_target` (default: 200) as`n*recalibration_target`.

Similar for `batch_size`